src/adzerk/boot_reload/reload.cljs

```diff
@@ -21,8 +21,8 @@
 (defn- changed-href? [href-or-uri changed]
```

Comment:
Hmm, I think this functions should be renamed. This is not a predicate but something like "select-changed-relative-uri-by-href-or-uri". Not sure if I can think of shorter name, but them long name doesn't hurt either. Docstring would be welcome also.